doc/DEPLOYMENT.rst

```diff
@@ -0,0 +1,229 @@
+
+Training your own language model is often a good way to improve transcription on your audio. The process and tools used to generate a language model are described in :ref:`scorer-scripts` and general information can be found in :ref:`decoder-docs`. Generating a scorer from a constrained topic dataset is a quick process and can bring significant accuracy improvements if your audio is from a specific topic.
+
+Acoustic model training is described in :ref:`training-docs`. Fine tuning an off-the-shelf acoustic model to your own data can be a good way to improve performance. See the :ref:`fine tuning and transfer learning sections <training-fine-tuning>` for more information.
+
+Model compatibility
+^^^^^^^^^^^^^^^^^^^
+
+🐸STT models are versioned to mitigate incompatibilities with clients and language bindings. If you get an error saying your model file version is too old for the client, you should either (1) upgrade to a newer model, (2) re-export your model from the checkpoint using a newer version of the code, or (3) downgrade your client if you need to use the old model and can't re-export it.
+
+.. _py-usage:
+
+Using the Python package
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pre-built binaries for deploying a trained model can be installed with ``pip``. It is highly recommended that you use Python 3.5 or higher in a virtual environment. Both `pip <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-pip>`_ and `venv <https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment>`_ are included in normal Python 3 installations.
+
+When you create a new python virtual environment, you create a directory containing a ``python`` binary and everything needed to run 🐸STT. For the purpose of this documentation, we will use on ``$HOME/coqui-stt-venv``, but you can use whatever directory you like.
+
+Let's make the virtual environment:
+
+.. code-block::
+
+   $ python -m venv $HOME/coqui-stt-venv/
+
+After this command completes, your new environment is ready to be activated. Each time you work with 🐸STT, you need to *activate* your virtual environment, as such:
+
+.. code-block::
+
+   $ source $HOME/coqui-stt-venv/bin/activate
+
+After your environment has been activated, you can use ``pip`` to install ``stt``, as such:
+
+.. code-block::
+
+   (coqui-stt-venv)$ python -m pip install stt
+
+After installation has finished, you can call ``stt`` from the command-line.
+
+The following command assumes you `downloaded the pre-trained models <#getting-the-pre-trained-model>`_.
```

Comment:
The correct way to reference sections in RST is by giving them a name, so that this link doesn't break if you change the section heading. See for example the `py-usage` section name above or `py-api-example` just below.